scipy/spatial/transform/rotation.py

```diff
@@ -281,3 +281,115 @@ def as_rotvec(self):
             return rotvec[0]
         else:
             return rotvec
+
+    @staticmethod
+    def _make_dcm_from_angle(char, angle, intrinsic=False):
+        sinx = np.sin(angle)
+        cosx = np.cos(angle)
+
+        axis = char.lower()
+        if axis == 'z':
+            dcm = np.array([
+                [cosx, -sinx, 0],
+                [sinx, cosx, 0],
+                [0, 0, 1]
+                ])
+        elif axis == 'y':
+            dcm = np.array([
+                [cosx, 0, sinx],
+                [0, 1, 0],
+                [-sinx, 0, cosx]
+                ])
+        elif axis == 'x':
+            dcm = np.array([
+                [1, 0, 0],
+                [0, cosx, -sinx],
+                [0, sinx, cosx]
+                ])
+
+        return dcm.transpose() if intrinsic else dcm
```

Comment:
For extrinsic/intrinsic rotations we don't need to transpose individual matrices, we only reverse the multiplication order. Think about it.